RoadRunner/src/main/java/com/acmerobotics/roadrunner/ftc/Encoders.kt

```diff
@@ -9,11 +9,11 @@ import kotlin.math.min
 import kotlin.math.round
 
 class PositionVelocityPair(
```

Comment:
I intend for this class to represent encoder measurements in tick units (and really measurements made through a Rev hub given the raw fields). I would rather create new types and plumbing for sensors that don't fit this model.

Reply:
Makes sense. It may still be worth having some way of expanding it for other ways of reading, such as Octoquad or the pinpoint's raw encoder passthrough system.